helm/hpcc/templates/thor.yaml

```diff
@@ -132,6 +132,7 @@ data:
 {{ include "hpcc.generateGlobalConfigMap" $ | indent 6 }}
 
 {{- if not $eclAgentUseChildProcesses }}
+{{- $eclAgentJobNamePrefix := printf "%s-%s-" .name  $eclAgentType }}
```

Comment:
as per previous comments, pod name should change to include target name (.name) too (line 140)